src/reversediff.jl

```diff
@@ -0,0 +1,235 @@
+const RTR = ReverseDiff.TrackedReal
+const RTV = ReverseDiff.TrackedVector
+const RTM = ReverseDiff.TrackedMatrix
+import SpecialFunctions: logbeta
+import Distributions: Gamma
+
+ReverseDiff.@forward binomlogpdf(n::Int, p::RTR, x::Int) = begin
```

Comment:
Maybe add some comments explaining what's happening here.

Reply:
Sure, this is using ForwardDiff for the function.